.github/workflows/eerelease.yml

```
@@ -71,11 +72,11 @@ jobs:
       - name: Build Community EE image
         working-directory: image/execution-environments/${{ inputs.ee-name }}
         run: |
-          ansible-builder build --tag "ghcr.io/ansible-community/${{ inputs.ee-name }}:${EE_VERSION}"
+          ansible-builder build --tag "ghcr.io/${{ env.IMAGE_NAME }}:${EE_VERSION}"
```

If you're using environment variables in a shell, please interpolate on shell level, not on GHA level.
```suggestion
          ansible-builder build --tag "ghcr.io/${IMAGE_NAME}:${EE_VERSION}"
```